pydust/src/pytypes.zig

```diff
@@ -80,6 +80,18 @@ fn Slots(comptime definition: type, comptime Instance: type) type {
 
     return struct {
         const methods = funcs.Methods(definition);
+        const binaryOperators = blk: {
```

Comment:
Why have this array? Vs just looping over later?

Reply:
Ah, leftover